src/main/java/io/github/thebusybiscuit/sensibletoolbox/utils/IntRange.java

```diff
@@ -0,0 +1,52 @@
+package io.github.thebusybiscuit.sensibletoolbox.utils;
+
+public class IntRange {
+
+    private final int min;
+    private final int max;
+
+    public IntRange(int number) {
+        this.min = number;
+        this.max = number;
+    }
+
+    public IntRange(int number1, int number2) {
+        super();
```

Comment:
This wont be doing anything?

Reply:
Basically yeah, as this object doest extend/implement anything, calling `super()` just calls `Object.super()` which is... nothing :D